### ISSUE TITLE
Not getting completions for function args

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#3389]
### Short Summary

You can see in the screenshot below that I'm not getting a competion for the argument `printdebug`.
![screenshot 2017-11-27 10 49 56](https://user-images.githubusercontent.com/655154/33283507-e3c0cf32-d360-11e7-9a1b-e50bd3098c8d.png)

### Steps to Reproduce

open *peSnippet.js*
go to line 261
If you need to, add space so you can start typing in the function `this.snippetInsertImpl`
Attempt to get completions for either `snippet` or `view.

### Expected results

completions for the args of the function

### Actual results

No completions for the args.

**Comment by cgchoffman** [comment]
@mitchell-as, any idea off the top of your head of complexity/time needed for fixing this?

**Comment by mitchell-as** [comment]
No, sorry. That should work though. I'd have to dig in order to figure out why that's not working. Perhaps it's the previous `printdebug.treat_as_ejs = true` assignment. If you delete that line do you get completions? If so, then this may not be an easy fix.

**Comment by cgchoffman** [comment]
If I use the example I give in text there is no followup assignment to the object in question and it still doesn't work for `snippet` which is an arg passed into `this.snippetInsertImpl`.  ie. there is no equivalent `printdebug.treat_as_ejs = true` in the text example and it's still not working so hopefully it's not an overly complicated fix.

**Comment by cgchoffman** [comment]
I just checked and this didn't work in Komodo 10 but apparently only in JS.  So labaled as enhancement.
